src/events/pollVotationResults.js

```diff
@@ -0,0 +1,14 @@
+const { Events } = require('discord.js');
+const { pollResults } = require('../utils/poll-vote');
+
+module.exports = {
+  name: Events.MessageCreate,
+  async execute(message) {
+
+    if (message.author.bot) {
+      return;
+    }
+
+    if (message.embeds[0]?.data?.type === 'poll_result') await pollResults(message)
```

Comment:
Maybe we can validate all we need on first conditional
 
```suggestion
    if (message.author.bot && message.embeds[0]?.data?.type !== 'poll_result') {
      return;
    }

    await pollResults(message)
```

Reply:
Complete